### PR TITLE
Adds some more (mostly medical related) tips

### PR DESCRIPTION
**Pull request by Absolucy**
## About The Pull Request

This adds some more tips to the game, mostly relating to medical, including some more newly added features.

## Why It's Good For The Game

Tips are good.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

it's a text file ffs

</details>

## Changelog
:cl:
add: Added some new tips (most of which are medical-related)
/:cl:

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -50,6 +50,7 @@ As a Medical Doctor, treating plasmamen is not impossible! Salbutamol stops them
 As a Medical Doctor, you can point your penlight at people to create a medical hologram. This lets them know that you're coming to treat them.
 As a Medical Doctor, you can extract implants without breaking them by holding an empty implant case in your offhand while performing the extraction step.
 As a Medical Doctor, clone scanning people will implant them with a health tracker that displays their vitals in the clone records. Useful to check on crew members that didn't activate suit sensors!
+As a Medical Doctor, you can always reference the wiki for blood type compatibility: https://wiki.beestation13.com/view/Guide_to_medicine#Bleeding
```
Comment (PowerfulBacon) [inline strings/tips.txt:53]: Do links work in tips?

File: strings/tips.txt [hunk 2]
```diff
@@ -273,11 +274,29 @@ Fun fact: reading this (https://beestation13.com/rules) may lead to an increase
...
-Death is normal. Chances are you'll get cloned, anyways.
+Death is normal. Chances are you'll get revived, anyways.
 Emping the AI has a chance to call the emergency shuttle.
 Beating revolutionaries over the head has a chance to deconvert them.
 You can examine someone to tell if their red eyes are natural or cultist.
 You can put holy water in darts to make them more effective against cultists. Or use the shotgun, cause it doesn't miss.
 Don't know what a word means? https://wiki.beestation13.com/view/Terminology
 You are able to block with most weapons! Being on any intent but harm and walking speed allows almost anything to block. More info: https://wiki.beestation13.com/view/Guide_to_Combat#Blocking
 Embedded items can be removed safely using a hemostat. If you don't have one on hand, try using a screwdriver or wirecutters; get creative!
+Whenever you have a corpse that's not on a stasis bed, jab them with an epi-pen, as they contain formaldehyde, which will halt organ decay.
```
Comment (PowerfulBacon) [inline strings/tips.txt:285]:
```suggestion
Whenever you have a corpse that's not on a stasis bed, jab them with an epi-pen. They contain formaldehyde, which will halt organ decay.
```

File: strings/tips.txt [hunk 3]
```diff
@@ -273,11 +274,29 @@ Fun fact: reading this (https://beestation13.com/rules) may lead to an increase
...
+You can use Cortex Imprint surgery, unlocked by Experimental Surgery tech, in order to render someone immune to mild brain traumas. This will not heal damage to the brain itself, however.
+Inacusiate will instantly heal all hearing damage whenever it is ingested, regardless of the amount ingested. A 0.1 unit pill will cure deafness just fine.
+The Mind Control nanite program, unlocked with Hazard Nanite Programs (which itself requires Alien Technology), completely bypasses mindshields, however it only lasts for 1 minute, and has a 3 minute cooldown before it can be triggered again.
+Hypnotic flashes will only induce a trance under certain conditions: the target must not be mindshielded, and they must either be asleep, hallucinating, suffering from reduced brain activity, or have low sanity.
+Advanced surgeries can only be started on an operating table. Don't try to start a revival surgery on a stasis bed, you won't see the option.
```
Comment (PowerfulBacon) [inline strings/tips.txt:292]: Isn't this dependant on the computer being present?

File: strings/tips.txt [hunk 4]
```diff
@@ -273,11 +274,29 @@ Fun fact: reading this (https://beestation13.com/rules) may lead to an increase
...
+Wound tending surgery works perfectly fine on IPCs. Don't question it.
+The brain traumas associated with hypnotic flashes and obsessions can only be cured through surgery — nanites and viruses will completely ignore them!
+You can free someone from an abductor-induced brainwashing by replacing the fleshy gland in their chest with a proper heart, and then performing a lobotomy on them.
+Changeling transformation stings can be reversed using clonexadone — it does not require a specific unit amount, and instead will be reversed after clonexadone has been in the victim's body for about a minute or so.
+When performing wound tending or blood filtering surgery, you can hold a health analyzer in your other hand in order to see how much damage is remaining on your patient.
+Oozelings can hug flaming people in order to help extinguish them.
+Trying to talk things out can often result in a better outcome than resorting to violence.
+Ghouls and voiceless dead (which can be identified as having a "blank, catatonic-like stare" when examined) will revert back to normal upon dying, restoring their appearance and freeing them from their loyalty to the heretic that converted them.
```
Comment (PowerfulBacon) [inline strings/tips.txt:301]: This tip gives confusing information as disconnected players have the same message. This encourages players to kill anyone with that message

File: strings/tips.txt [hunk 5]
```diff
@@ -273,11 +274,29 @@ Fun fact: reading this (https://beestation13.com/rules) may lead to an increase
...
+You can use Cortex Imprint surgery, unlocked by Experimental Surgery tech, in order to render someone immune to mild brain traumas. This will not heal damage to the brain itself, however.
+Inacusiate will instantly heal all hearing damage whenever it is ingested, regardless of the amount ingested. A 0.1 unit pill will cure deafness just fine.
+The Mind Control nanite program, unlocked with Hazard Nanite Programs (which itself requires Alien Technology), completely bypasses mindshields, however it only lasts for 1 minute, and has a 3 minute cooldown before it can be triggered again.
+Hypnotic flashes will only induce a trance under certain conditions: the target must not be mindshielded, and they must either be asleep, hallucinating, suffering from reduced brain activity, or have low sanity.
+Advanced surgeries can only be started on an operating table. Don't try to start a revival surgery on a stasis bed, you won't see the option.
+Don't overuse bruise packs or ointment on people, as these contain styptic powder and silver sulfadiazine respectively, which are capable of overdosing someone and causing suffocation damage if overused!
```
Comment (PowerfulBacon) [inline strings/tips.txt:293]: Overused is used twice here

File: strings/tips.txt [hunk 6]
```diff
@@ -273,11 +274,29 @@ Fun fact: reading this (https://beestation13.com/rules) may lead to an increase
...
+There are multiple ways to make surgery steps take less time. Operating tables will provide a speed boost compared to stasis beds (and alien operating tables even more so), surgery on sleeping or dead patients will be faster than awake patients, and various reagents such as honey, sterilizer spray, or alcohol can be applied after a surgery is started to provide a minor speed boost.
+You can use Cortex Imprint surgery, unlocked by Experimental Surgery tech, in order to render someone immune to mild brain traumas. This will not heal damage to the brain itself, however.
+Inacusiate will instantly heal all hearing damage whenever it is ingested, regardless of the amount ingested. A 0.1 unit pill will cure deafness just fine.
+The Mind Control nanite program, unlocked with Hazard Nanite Programs (which itself requires Alien Technology), completely bypasses mindshields, however it only lasts for 1 minute, and has a 3 minute cooldown before it can be triggered again.
+Hypnotic flashes will only induce a trance under certain conditions: the target must not be mindshielded, and they must either be asleep, hallucinating, suffering from reduced brain activity, or have low sanity.
```
Comment (PowerfulBacon) [inline strings/tips.txt:291]: The commas seem off in this one

File: strings/tips.txt (hunk 4 above)
Comment (Absolucy) [reply strings/tips.txt:301]: Hm, fair. I was trying to word it literally, but fair point:
```dm
/datum/status_effect/ghoul
	id = "ghoul"
	status_type = STATUS_EFFECT_UNIQUE
	duration = -1
	examine_text = "<span class='warning'>SUBJECTPRONOUN has a blank, catatonic like stare.</span>"
```

**Pull request closed without merging**